README.md

```diff
@@ -43,6 +43,7 @@ Before you start working with pgai, you need:
 
 * An [OpenAI API Key](https://platform.openai.com/api-keys) - everyone
 * A postgres client like [psql v16](https://docs.timescale.com/use-timescale/latest/integrations/query-admin/psql/) or [PopSQL](https://docs.timescale.com/use-timescale/latest/popsql/)
+* [Docker](https://docs.docker.com/get-docker/) - if you prefer to use pgai locally
 * [Python3](https://www.python.org/downloads/) - if you prefer code to pure sql 
 
 ## Create a pgai environment
```

Comment:
```suggestion
## Installation
```

Reply:
First thing I look for in a new-to-me repo as a developer/user is an "Installation" section.